userguide/tutorials/using-https.adoc

```diff
@@ -1,5 +1,7 @@
 = Using HTTPS
```

Comment:
Is this file still in use? I couldn't find where it's linked on the left? Also, is it different than https://docs.killbill.io/latest/how-to-add-a-certificate-using-ACM ?

Maybe it's related to the clean-up work from Jim (https://github.com/killbill/killbill-docs/pull/555)? Would be good to close the loop on that PR too.

Reply:
> Maybe it's related to the clean-up work from Jim (https://github.com/killbill/killbill-docs/pull/555)

Yes, it is possible, I'll double-check if this file is still in use.